app/login.tsx

```diff
@@ -78,13 +80,40 @@ const Login: React.FC = () => {
       setLoading(false);
     }
   };
+  async function checkUrl(url: string) {
+    // remove trailing / so we don't double add double / in my checker
+    if (url.endsWith("/")) {
+      url = url.slice(0, -1);
+    }
 
-  const handleConnect = (url: string) => {
+    if( await fetch("https://" + url + "/web/#/home.html", {mode: 'cors'}).then((response) => {
```

Comment:
Would it not be better to hit the server info API endpoint? It's un authenticated and also works for jellyfin installations without the webui.

Reply:
genuinely just didn't think of the /System/Info/Public endpoint being a thing :laughing: 

/System/Info/Public seems to work, thanks for the comment!